site/en/docs/extensions/mv3/intro/mv3-migration/index.md

```diff
@@ -233,255 +262,340 @@ Replace the following:
 
-The blocking version of the [webRequest](/docs/extensions/reference/webRequest)
-API still exists in Manifest V3 but its use is restricted to force-installed extensions
-only. See Chrome Enterprise policies:
-[ExtensionSettings](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ExtensionSettings),
-[ExtensionInstallForcelist](https://cloud.google.com/docs/chrome-enterprise/policies/?policy=ExtensionInstallForcelist).
+The blocking version of the [webRequest][api-webrequest] API still exists in Manifest V3 but its use
+is restricted to force-installed extensions only. See Chrome Enterprise policies:
+[ExtensionSettings][enterprise-settings], [ExtensionInstallForcelist][enterprise-force-list].
 
-All other extensions must now use
-[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest) for
-network request modification. This moves the actual modification of the network
-request into the Chrome browser: the extension no longer can read the actual
-network request, and in most cases needs no host permissions.
+All other extensions must now use [declarativeNetRequest][api-declarativenetrequest] for network
+request modification. This moves the actual modification of the network request into the Chrome
+browser: the extension no longer can read the actual network request, and in most cases needs no
+host permissions.
+
+{% Aside 'caution' %}
 
-{% Aside %}
 Request redirects and header modifications **do** require the user to grant host permissions.
-{% endAside %}
 
+{% endAside %}
 
 ### How do you use declarativeNetRequest?  {: #how-use-declarativenetrequest }
 
-Instead of reading the request and programmatically altering it, your extension
-specifies a number of rules, which map a set of conditions to corresponding
-actions. See the
-[declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest)
-reference documentation for a more detailed description of rules.
+Instead of reading the request and programmatically altering it, your extension specifies a number
+of [rules][api-declarative-rules], which map a set of conditions to corresponding actions. See the
+[declarativeNetRequest][api-declarativenetrequest] reference documentation for a more detailed
+description of rules.
 
-This feature allows content blockers and other request-modifying extensions to
-implement their use cases without requiring host permissions, and without
-needing to read the actual requests.
+This feature allows content blockers and other request-modifying extensions to implement their use
+cases without requiring host permissions, and without needing to read the actual requests.
 
 {% Aside %}
-In order to aid with the migration process, the declarativeNetRequest API is
-available for use in Manifest V2 extensions as of Chrome 84.
+
+In order to aid with the migration process, the declarativeNetRequest API is available for use in
+Manifest V2 extensions as of Chrome 84.
+
 {% endAside %}
```

Comment:
This aside feels very out of place here. We definitely should talk about how to effectively use the `"minimum_chrome_version"` field and some key Chrome versions developers might want to target, but this doesn't seem like the right place for it.